include/zephyr/net/buf.h

```diff
@@ -884,15 +884,6 @@ static inline void net_buf_simple_restore(struct net_buf_simple *buf,
 	buf->len = state->len;
 }
 
-/**
- * Flag indicating that the buffer has associated fragments. Only used
- * internally by the buffer handling code while the buffer is inside a
- * FIFO, meaning this never needs to be explicitly set or unset by the
- * net_buf API user. As long as the buffer is outside of a FIFO, i.e.
- * in practice always for the user for this API, the buf->frags pointer
- * should be used instead.
- */
-#define NET_BUF_FRAGS        BIT(0)
```

Comment:
there is some 'illegal' usage of this flag in `subsys/bluetooth/mesh/friends.c`

Reply:
@jhedberg @PavelVPV will you fix this or should I?

Reply:
If it doesn't take much of your time, could please fix it? I can look at this, but early next week.

Reply:
Actually, I can look at this on Monday because I'm anyway going to work in `subsys/bluetooth/mesh/friend.c` next week.

Reply:
OK! If I don't manage to look at this today I will leave it to you on Monday, thanks @PavelVPV.